packages/list-plugins/lib/tracking/byTracking.js

```diff
@@ -0,0 +1,70 @@
+const { Relationship } = require('@keystone-alpha/fields');
+
+const composeResolveInput = (originalHook, newHook) => async params => {
+  const resolvedData = await originalHook(params);
+  return newHook({ ...params, resolvedData });
+};
+
+const _byTracking = ({ created = true, updated = true }) => ({
+  updatedByField = 'updatedBy',
+  createdByField = 'createdBy',
+  trackingListName = 'User',
+  ...byFieldOptions
+} = {}) => ({ fields = {}, hooks = {}, ...rest }) => {
+  const relationshipOptions = {
+    type: Relationship,
+    ref: trackingListName,
```

Comment:
We don't need this. We spread `byFieldOptions` so they can just pass `ref`

Reply:
done